src/btree/bt_stat.c

```diff
@@ -126,6 +126,47 @@ __stat_page(WT_SESSION_IMPL *session, WT_PAGE *page, WT_DSRC_STATS **stats)
     return (0);
 }
 
+/*
+ * __stat_page_col_fix --
+ *     Stat a WT_PAGE_COL_FIX page.
+ */
+static void
+__stat_page_col_fix(WT_SESSION_IMPL *session, WT_PAGE *page, WT_DSRC_STATS **stats)
+{
+    WT_CELL *cell;
+    WT_CELL_UNPACK_KV unpack;
+    WT_INSERT *ins;
+    uint32_t tw, numtws, stat_entries, stat_tws;
```

Comment:
Fixed.
(To me, tw and numtws go together because one is an iteration variable and the other is its bound; therefore they should be declared together. So it's always going to come out that way and I'm not necessarily going to notice later either. So, unfortunately, this is going to keep happening.)